plugin.video.srfplaytv/resources/settings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
can you please use the new settings format?

Reply:
Do I just need to replace `UTF-8` by `utf-8` or do you mean something else?

Reply:
Do you mean this: https://kodi.wiki/view/Add-on_settings_conversion ?